js/apps/system/_admin/aardvark/APP/frontend/js/routers/router.js

```diff
@@ -61,8 +61,7 @@
       'logs': 'logger',
       'helpus': 'helpUs',
       'views': 'views',
-      'view/:name': 'viewSettings',
-      'view/:name/*link': 'viewSettings',
+      'views/:name': 'views',
```

Comment:
exisiting URLs are `view/viewName`; with this change it'll be `views/viewName` (`views` instead of `view`). This will cause old URLs to break. 
1. Is that an intended change?
2. If yes, we should probably add redirects from older URLs to new ones?